common/constants/custom_panels.ts

```diff
@@ -9,7 +9,21 @@
  * GitHub history for details.
  */
 
-export const CUSTOM_PANELS_API_PREFIX = '/api/observability/custom_panels';
+export const CUSTOM_PANELS_API_PREFIX = '/api/observability/operational_panels';
 export const CUSTOM_PANELS_DOCUMENTATION_URL = 'https://www.opensearch.org';
 export const CREATE_PANEL_MESSAGE = 'Enter a name to describe the purpose of this custom panel.';
-export const DATE_FORMAT = 'MM/DD/YYYY hh:mm A';
+export const DATE_FORMAT = 'YYYY-MM-DD HH:mm:ss';
+export const INDEX_REGEX = /(search source|source|index)\s*=\s*([^|\s]+)/i;
```

Comment:
this is also used by Eric, maybe move this to shared and change his function to use it as well?

Reply:
Will move it to this file https://github.com/opensearch-project/trace-analytics/blob/feature/observability/common/constants/shared.ts later.